gitlint/__init__.py

```diff
@@ -44,6 +44,8 @@
 import os
 import os.path
 import sys
+from concurrent import futures
+from multiprocessing import cpu_count
```

Comment:
could you change this to just import the multiprocessing module. I follow the google style guide in that regards. https://google.github.io/styleguide/pyguide.html#Imports

Reply:
Done.